.gitignore

```diff
@@ -1 +1,3 @@
+.direnv
 /target
+result
```

Comment:
What's in result? Nix creates it?

Reply:
That's a symlink to the derivation produced by `nix build`!